app/javascript/apiService.js

```diff
@@ -77,7 +77,7 @@ const fetchApplications = async ({ page, filters }) =>
     true
   )
 
-const fetchLeaseUpApplications = async (listingId, page, { filters }) => {
+const fetchLeaseUpApplications = async (listingId, page, { filters }, withGeneral = true) => {
```

Comment:
Not that I need to understand it, but `withGeneral` isn't immediately clear.  Maybe `includeGeneralApps`?  But I'm not sure what a "general application" is.

Reply:
Good call! It's the applications without preferences - they don't need to be called for processing layered validations. I'll update the parameter name and add a code comment